lib/luminork-server/src/service/v1/components/mod.rs

```diff
@@ -450,6 +462,9 @@ pub struct ComponentViewV1 {
     pub connections: Vec<ConnectionViewV1>,
     // what views this component is in
     pub views: Vec<ViewV1>,
+    // Any connected secret
```

Comment:
I would say something to the effect of "the ID of the secret, if this is a secret-defining component and a secret has been set" or something that sounds better